cms/djangoapps/contentstore/views/videos.py

```diff
@@ -468,3 +482,35 @@ def is_status_update_request(request_data):
     Returns True if `request_data` contains status update else False.
     """
     return any('status' in update for update in request_data)
+
+def _get_validated_params(request):
+    """
+    Reads and returns valid parameters
```

Comment:
Isn't there this kind of method present in the edx-platform ? this looks generic to pagination, if there is already one present use it, otherwise we would have to place this in some generic location.